src/main/java/org/irenical/drowsy/query/BaseQuery.java

```diff
@@ -37,6 +39,15 @@ public void setQuery(String query) {
     this.query = query;
   }
 
+  public void setQueryFromResource(String resourceFilePath) {
+    InputStream is = this.getClass().getResourceAsStream(resourceFilePath);
```

Comment:
Overload this method with a setQueryFromResource(ClassLoader, String) as the resource might not be available in Drowsy's class ClassLoader